### PR TITLE
Make encrypted group operation recommendation conditional.

### PR DESCRIPTION
**Pull request by Bren2010**

### CODE REVIEW

**Comment by ekr** [comment]
As noted in the discussion about issue #246, I don't think this is the right answer.

File: draft-ietf-mls-architecture.md [hunk 1]
```diff
@@ -915,8 +915,10 @@ While the Application messages will always be encrypted, having the handshake
 messages in plaintext has privacy consequences as someone could
 collect the signatures on the handshake messages and use them for tracking.
 
-> **RECOMMENDATION:** Prefer using encrypted group operation messages to avoid
-> privacy issues related to non-encrypted signatures.
+> **RECOMMENDATION:** Prefer using encrypted group operation messages unless a
+> Delivery Service explicitly needs plaintext group operations to provide
```
Comment (raphaelrobert) [inline draft-ietf-mls-architecture.md:919]:
```suggestion
> Delivery Service explicitly needs plaintext handshake messages to provide
```

**Review by rohanmahy — APPROVED** [review]
If we go with this PR, I prefer using "handshake" messages as Raphael suggests.

**Review by beurdouche — APPROVED** [review]

**Comment by ekr** [comment]
Please back this out. I wasn't at the interim and there wasn't a consensus call on the list, so we shouldn't be landing anything until that happens.

**Comment by seanturner** [comment]
Yeah we should back this out.  I'll set something up to finalize this.

**Pull request merged**